### ISSUE TITLE
mousewheel (assignable as shortcut) to change frames in timeline

### ISSUE THREAD

**Issue by awesomez** [aseprite/aseprite#1789]
I have become accustomed to Graphics Gale's mousewheel controls (mousewheel up/down) to change frames during animation.  No matter how much I love Aseprite's overall functionality -- I can't bring myself to animate in it. 

And that's a real problem -- I'm primarily a pixel animator (NOT a pixel-artist), so this really sucks for me.

To give an idea of how it works, Graphics Gale lets you use special keys (i.e. F10, F11, F12) as "keys" to assign "keyboard-shortcuts" for mouse-controls, particularly **mouse-_wheel_ shortcut controls** (i.e. middlemouse, mousewheel-up, and mousewheel-down, respectively).

Would you be willing to drop this functionality into the next version of Aseprite -- pretty-please?? -- I've been really wanting to use it for this purpose, but I just can't bring myself to it. Graphics Gale is still so much better for animation, but the pixel-art tools are infinitely better in Aseprite.

 Please help. D:

### Aseprite and System version

* Aseprite version: version number, installer/portable/Steam/beta/dev/commit-hash
* System: Windows/macOS/Linux, version, distribution

**Comment by duckonst1** [comment]
Helo

**Comment by dacap** [comment]
There are plans to add customization for the mouse wheel (#680, #970), I'll try to add this for v1.2.9.

**Comment by awesomez** [comment]
Thanks a ton man! -- Moving between frames using the mousewheel would be an awesome addition!

@dacap is da-man!!

Now if I could only right-click + drag to temporarily select a portion of my sprite with a lasso or box tool, then rotate that portion of my sprite (being able to use "rotsprite"), we'd be cooking with fire~!! 🔥 

That's really the only two productivity things kicking me in the teeth right now in terms of animation with Aseprite!

**Comment by dacap** [comment]
Actually I was just testing and you can use Ctrl+mouse wheel to move between frames.

**Comment by awesomez** [comment]
Right, but I don't usually zoom when animating -- I keep a steady zoom the entire time I do animation. Scrolling frames would be infinitely more useful with the mousewheel -- using Ctrl+mousewheel to change zoom levels would be better at that point.

Either that, or allow me to zoom in the main panel with the mousewheel and scroll frames when rolling the wheel in the timeline area -- or maybe both options?

**Comment by golddotasksquestions** [comment]
I know this is not the same workflow you guys requested, but it gives me similar results. I use Controls similar as they are in Photoshop:
My left hand on the keyboard and use A and D key to move between frames, Space for the Hand-tool (quick) and Ctrl+Space+ leftmouse for Zoom-tool(quick). 
If you are a gamer, A and D feels very natural. And with my right hand, I work with a tablet, so I usually don't have the scroll wheel available.

**Comment by awesomez** [comment]
This really depends on your hands and the shape of your fingers as to whether this feels "natural" -- I'm a gamer, but standard keyboard controls have never once felt more "natural" to me than using the mouse.

That being said, being able to assign a keyboard shortcut isn't a bad idea, but as a pixel animator who rarely uses a tablet, I prefer the mouse options.
